### DISCUSSION TITLE
Question about adapter documentation

### DISCUSSION THREAD

**Discussion by nbarrientos** [Koenkk/zigbee2mqtt#21345] [Devices]
Hi,

First of all many thanks for maintaining this project, I've been using it daily since 2019 and also modestly sponsoring it for a while now. 

I'm one of the old farts still using a CC2531 adapter (IIRC this was one of the first adapters supported, if not the first). However, the family has grown and I've faced a few times a clogged coordinator (the infamous `Error: SRSP - ZDO - mgmtPermitJoinReg` error message), forcing me to pay a visit to the rack to unplug and plug the adapter. Reading on it seems that one way to address this problem is to get a more modern dongle with a better chip. Fair enough, time to upgrade!

Reading the documentation there seems to be [an upgrade path ](https://www.zigbee2mqtt.io/guide/faq/#what-does-and-does-not-require-repairing-of-all-devices)for me to a CC2652 device without having to re-pair everything, by flashing the IEEE addr of the old device into the new one (great!). After reading this I decided to go ahead and select a new USB dongle, out of the options in the [list of recommended adapters](https://www.zigbee2mqtt.io/guide/adapters/#based-on-texas-instruments-cc2652-cc1352-chip). 

I like the Slaesh's CC2652RB stick, it fits my requirements and I can source it rather easily from my location. However, reading the docs of the manufacturer it [states](https://slae.sh/projects/cc2652/#zigbee2mqtt):

> When migrating from another dongle (e.g. CC2531), make sure to modify your pan_id in your [configuration](https://www.zigbee2mqtt.io/information/configuration.html), otherwise Zigbee2mqtt won’t start. If there is no entry pan_id, please create one using any other number than 0x1a62! This is the default one, if there is no one setup. You have to re-pair your devices then, cause we created a new network.

which puts me off because re-pairing everything would be lots of work.

I'd be very grateful if somebody with experience with this device and a migration from CC2531 could confirm if the documentation is accurate and/or or it's possible to jump into this device without having to re-pair. If that's not possible, could anybody recommend one device out of the ones in the list that could "guarantee" a smooth migration?

/cc @slaesh

Thanks!

**Comment by Koenkk** [comment]
See https://www.zigbee2mqtt.io/guide/faq/#requires-repairing

TLDR; no repairing needed.

**Reply by nbarrientos** [reply]
Thanks. So maybe the documentation on the website of the dongle should be patched to avoid confusion. At least it confused me :smile:

**Reply by Koenkk** [reply]
I agree, but I do not control that site (contact the owners directly)

**Reply by nbarrientos** [reply]
@slaesh patched it already, thanks both!
